xbmc/settings/dialogs/GUIDialogSettingsBase.cpp

```diff
@@ -521,13 +521,20 @@ std::set<std::string> CGUIDialogSettingsBase::CreateSettings()
     const CSettingControlTitle *title = dynamic_cast<CSettingControlTitle *>((*groupIt)->GetControl());
     bool hideSeparator = title ? title->IsSeparatorHidden() : false;
```

Comment:
my PR could probably be simplified by changing this line to:
bool hideSeparator = title ? title->IsSeparatorHidden() : first;

but I wasn't sure about it, because the presence of "title" probably doesn't mean that a label is set as well. @Montellese ?